apiserver/client/status.go

```diff
@@ -533,21 +635,50 @@ func makeMachineStatus(machine *state.Machine) (status params.MachineStatus) {
 		}
 		status.DNSName = addr.Value
 
-		mAddrs := machine.Addresses()
```

Comment:
I think we probably need to keep the old IPAddresses field for backwards compatibility?